.rspec

```diff
@@ -0,0 +1,2 @@
+--color
+--require spec_helper
```

Comment:
I could add additional format system like proggres bar `Fuubar`, but also i could define some stock formats like documentation. I've never used it then it could be some new experience ;)